wolfcrypt/src/asn.c

```diff
@@ -8870,6 +8899,134 @@ static int SetAltNames(byte *out, word32 outSz, byte *input, word32 length)
 }
 #endif /* WOLFSL_ALT_NAMES */
 
+/* Encodes one attribute of the name (issuer/subject)
+ *
+ * name     structure to hold result of encoding
+ * nameStr  value to be encoded
+ * nameType type of encoding i.e CTC_UTF8
+ * type     id of attribute i.e ASN_COMMON_NAME
+ *
+ * returns length on success
+ */
+static int _EncodeName(EncodedName* name, const char* nameStr, char nameType,
```

Comment:
Let's avoid _ prefix as some compilers reserver that namespace.

Reply:
Thanks for the review added a commit to fix/change the items.